_shows/72_73/tis_pity_shes_a_whore.md

```diff
@@ -3,12 +3,84 @@ title: "'Tis Pity She's A Whore"
 season: In House
 period: Spring
 playwright: John Ford
-season_sort: 130
+season_sort: 131
 venue: New Theatre
 
 cast:
- - role: Vazquez
-   name: Tim Sluckin
+- role: Annabella
+  name: Kay Jackaman
+- role: Giovanni
+  name: Tony Buckby
+  note: her brother
+- role: Vazquez
+  name: Tim Sluckin
+- role: Florio
+  name: Bill Simpson
+  note: their father
+- role: Soranzo
+  name: Kevin Buxton
+- role: Friar Boneventure
+  name: Stuart Mitchell
+- role: Putana
+  name: Janice Haslam
+- role: Richardetto
+  name: Stephen Dixon
+  note: a false doctor
+- role: Hippolyta
+  name: Susan Landon
+  note: his wife, a supposed widow
+- role: Philotis
+  name: Jane Leah
+  note: his niece
+- role: Donado
+  name: Steve Perry
+  note: a rich magnifico
+- role: Bergetto
+  name: Chrs Button
+  note: his nephew
+- role: Poggio
+  name: Phil Barnes
+  note: servent to Bergetto
+- role: Grimaldi
+  name: John Ironmonger
+  note: a soldier
+- role: Cardinal
+  name: Martin Holland
+- role: Officer / Bandit
+  name: Kevin Ashcroft
+- role: Masquer
+  name: Celia Evans
+- role: Masquer
+  name: Janice Howman
+- role: Masquer
+  name: Julia Wykes
+
+crew:
+- role: Director
+  name: John McRae
+- role: Designer
+  name: Stuart Mitchell
+- role: Production Assistant
+  name: Susan P. Landon
+- role: Stage Manager
+  name: Hilary Renwick
+- role: Prompt
+  name: Kathy Hodgetts
+- role: Lighting
+  name: Adam Bent
+- role: Wardrobe
+  name: Julia Taylor
+- role: Wardrobe
+  name: Diane Thale
+- role: Sound
+  name: Jeremy Hunter
+- role: Organist
+  name: Derek Carden
+- role: Fights
+  name: John Ironmonger
+- role: Front-Of-House Manger
+  name: Janet Bell
+
 ---
 
 Tis Pity She's a Whore is a tragedy written by John Ford. It was likely first performed between 1629 and 1633.
```

Comment:
The title of the show should be in single quotes, and is missing an apostrophe before 'Tis'.